server/src/test/java/org/elasticsearch/index/mapper/TextFieldMapperTests.java

```diff
@@ -249,6 +251,61 @@ public void testDefaults() throws IOException {
         assertEquals(DocValuesType.NONE, fieldType.docValuesType());
     }
 
+    public void testStoreParameterDefaults() throws IOException {
```

Comment:
I think this test needs to be adjusted with the serialization logic removed to address the CI failure?